test/locale.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { classes } from "../src/commands";
+import { COMMAND_LOCALIZATIONS, loadTranslations } from "../src/locale";
+
+describe(".po files", () => {
+	test("syntax is valid", () => {
+		const locales = loadTranslations();
+		expect(locales.length).toBe(COMMAND_LOCALIZATIONS.length);
+		for (const locale of COMMAND_LOCALIZATIONS.map(l => l.gettext)) {
+			expect(locales).toContain(locale);
+		}
+	});
+	test("command metadata is valid", () => {
+		loadTranslations();
+		const commands = classes.map(command => command.meta);
+		let translationsLoaded = false;
+		for (const command of commands) {
+			if (command.description_localizations?.ko !== command.description_localizations?.ja) {
```

Comment:
Unclear what the purpose of this check is. They're both localisations we expect to exist and we're checking that neither has fallen back to a default or is undefined? Unless the test is literally "the KO and JA localisations aren't identical" which seems strange, would prefer explicit checks for what we're checking or at least a comment to explain.

Reply:
This is to verify that the command metadata actually contains our localisations, since there was a time when they didn't and the "localisations" were actually all the English text.